pkg/kubeapiserver/admission/initializer.go

```diff
@@ -163,15 +136,15 @@ func (i *PluginInitializer) Initialize(plugin admission.Interface) {
 		wants.SetInternalKubeClientSet(i.internalClient)
 	}
 
-	if wants, ok := plugin.(WantsExternalKubeClientSet); ok {
+	if wants, ok := plugin.(admissioninit.WantsExternalKubeClientSet); ok {
```

Comment:
Seems like these blocks can be removed.

Reply:
Actually it is more than that. The whole `kubeapiserver. PluginInitializer` deserves a little more attention. I'd like to follow up that.

Reply:
> Actually it is more than that. The whole kubeapiserver. PluginInitializer deserves a little more attention. I'd like to follow up that.

ok